src/datadog-step-functions.ts

```diff
@@ -155,6 +158,28 @@ export class DatadogStepFunctions extends Construct {
       )
     );
   }
+
+  private setTags(stateMachine: sfn.StateMachine): void {
```

Comment:
Modified from https://github.com/DataDog/datadog-cdk-constructs/blob/main/src/datadog-lambda.ts#L181-L205. Not reusing it because the tag keys are different (e.g. `env` vs `DD_ENV`).